src/manifest/mod.rs

```diff
@@ -26,6 +26,10 @@ pub struct CrateData {
 #[derive(Deserialize)]
 struct CargoManifest {
     package: CargoPackage,
+
+    #[serde(default)]
```

Comment:
I think I might need to put this on all the members below too, so that if you just define a profiling config and not a dev and release config, it doesn't break deserializing...